tests/coverage/unreachable/wrong_coverage_1/main.rs

```diff
@@ -0,0 +1,13 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+#[kani::proof]
+fn wrong_coverage_1() {
```

Comment:
Let's have a more appropriate name for this function now, it's not wrong anymore!